addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -678,6 +678,11 @@ export function getTraversedNodes(editable, range = getDeepRange(editable)) {
         node = iterator.nextNode();
         node && traversedNodes.add(node);
     }
+    if (node) {
+        for (const descendant of descendants(node)) {
+            traversedNodes.add(descendant);
+        }
+    }
```

Comment:
I don't this this is right. Consider this:
```html
<h1>a[bc</h1>
<p>]def</p>
```
At the end of the loop, `node` is the P which is the end container. After that the suggested change will add `"def"` to the set of traversed nodes, which is wrong.